website/blog/2023-03-17-databend-weekly-85.md

```diff
@@ -0,0 +1,186 @@
+  - name: Carlosfengv
+  - name: Chasen-Zhang
+  - name: dantengsky
+  - name: dependabot[bot]
+  - name: drmingdrmer
+  - name: everpcpc
+  - name: jun0315
+  - name: leiysky
+  - name: lichuang
+  - name: mergify[bot]
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: wubx
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+> :loudspeaker: Read our blog *[Way to Go: OpenDAL successfully entered Apache Incubator](https://databend.rs/blog/opendal-enters-apache-incubator)* to learn about the story of [OpenDAL](https://github.com/apache/incubator-opendal).
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Data Type: MAP
+
+The MAP data structure holds `Key:Value` pairs using a nested `Array(Tuple(key, value))` and is useful when the data type is constant but the Key's value cannot be fully determined. The Key must be of a specified basic data type and duplicates are not allowed, while the Value can be any data type including nested arrays or tuples. A bloom filter index is created in Map makes it easier and faster to search for values in MAP.
+
+```sql
+select * from nginx_log where log['ip'] = '205.91.162.148';
++----+----------------------------------------+
+| id | log                                    |
++----+----------------------------------------+
+| 1  | {'ip':'205.91.162.148','url':'test-1'} |
++----+----------------------------------------+
+1 row in set
+```
+
+If you want to learn more about the Map data type, please read the following materials:
+
+- [Docs | Data Types - Map](https://databend.rs/doc/sql-reference/data-types/data-type-map)
+
+### Data Transformation During Loading Process
+
+Do you remember the two RFCs mentioned last week? Now, Databend has added support for data transformation during the loading process into tables. Basic transformation operations can be achieved by using the `COPY INTO <table>` command.
+
+```sql
+CREATE TABLE my_table(id int, name string, time date);
+
+COPY INTO my_table
+FROM (SELECT t.id, t.name, to_date(t.timestamp) FROM @mystage t)
+FILE_FORMAT = (type = parquet) PATTERN='.*parquet';
+```
+
+This feature avoids storing pre-transformed data in temporary tables and supports column reordering, column omission, and type conversion operation. In addition, partial data can be loaded from staged Parquet files or their columns can be rearranged. This feature simplifies and streamlines ETL processes, allowing users to focus on data analysis rather than mechanically moving it.
```

Comment:
allowing users to focus on data analysis rather than mechanically moving it.--> allowing uses to give more attentions on the data analysis without considering how to move their data.